src/flash/com/longtailvideo/jwplayer/media/VideoMediaProvider.as

```diff
@@ -50,6 +50,10 @@ public class VideoMediaProvider extends MediaProvider {
     /** Is buffering due to load/seek or underflow? **/
     private var seeking:Boolean;
 
+    private var _preload:Boolean;
```

Comment:
I tried to find a way to remove flags, but some functions in here (e.g. loadStream) need to know whether if we are preloading while they do not have any information about the item. Do you have any suggestions?